app/router.go

```diff
@@ -75,8 +75,13 @@ func Router(c collector) *mux.Router {
 	get.MatcherFunc(URLMatcher("/api/origin/host/{id}")).HandlerFunc(
 		gzipHandler(makeOriginHostHandler(c)))
 	get.HandleFunc("/api/report", gzipHandler(makeRawReportHandler(c)))
+	get.HandleFunc("/api/controls", controlRouter.HandleProbeWS)
 	get.PathPrefix("/").Handler(http.FileServer(FS(false))) // everything else is static
 
+	post := router.Methods("POST").Subrouter()
+	post.HandleFunc("/api/report", makeReportPostHandler(c)).Methods("POST")
+	post.MatcherFunc(URLMatcher("/api/control/{probeID}/{nodeID}/{control}")).HandlerFunc(
+		controlRouter.HandleControl)
```

Comment:
Might separate concerns a bit to invert the registration? Not sure...

```Go
controlRouter.RegisterRoutes(router.PathPrefix("/api/control").Subrouter())
```
or even
```Go
NewControlRouter(router.PathPrefix("/api/control").Subrouter())
```